pycheribuild/config/chericonfig.py

```diff
@@ -278,6 +278,8 @@ def __init__(self, loader, action_class):
                                                              help="Attach to the same container again (note: "
                                                                   "docker-container option must be an id rather than "
                                                                   "a container name")
+        self.docker_uid = loader.add_option("docker-uid", help="UID to under Docker", default="", group=loader.docker_group)
```

Comment:
Missing a word? Also why "" rather than None? And could you not just default this to os.get[ug]id() so you don't need to check whether it's set?